backend/app/core/config.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import yaml
+from pydantic import BaseSettings, RedisDsn, validator
+
+
+class RedisSettings(BaseSettings):
+    uri: RedisDsn
+
+
+class ServerSessionsSettings(BaseSettings):
+    session_key: str
+    expires_in: int
+
+
+class Settings(BaseSettings):
+
+    redis: RedisSettings
+    server_sessions: ServerSessionsSettings
+
+    @classmethod
+    def from_yaml(cls, path_to_file):
+        absolute_path_to_file = Path().absolute() / path_to_file
+
+        try:
+            with open(str(absolute_path_to_file)) as f:
+                yaml_settings = yaml.safe_load(f)
+                return Settings.parse_obj(yaml_settings)
+        except (IOError, ImportError) as err:
+            print(f"Couldn't load config from file. Error: {err}")
+
+
+settings = Settings.from_yaml("config.yml")
```

Comment:
We probably want to make this configurable via a command line argument or something at some point (doesn't have to be now)